lib/ansible/modules/storage/purestorage/purefa_user.py

```diff
@@ -44,10 +44,11 @@
     description:
     - If changing an existing password, you must provide the old password for security
     type: str
-  api_token:
+  api:
     description:
     - Define whether to create an API token for this user
     - Token can be exposed using the I(debug) module
+    version_added: '2.9'
```

Comment:
please remove version_added because this is a bug fix and will be backported to 2.8. I know the build willl fail but you can ignore this.

Reply:
Thank you. I was uncomfortable with having to put this in to make the build pass.